### ISSUE TITLE
Validating publisher/signer of signed image in Docker/Notary (Docker Content Trust)

### ISSUE THREAD

**Issue by gwaines** [distribution/distribution#3607]
Need a mechanism via 'docker trust' or notary APIs/CLIs to validate that a particular signer has signed a particular image.
e.g.

$ docker trust inspect **--signer gwaines.pub** admin-2.cumulus.wrs.com:30093/wrcp-gwaines/hellokitty:v1.0
// which would validate that the hellokitty:v1.0 image is signed by the gwaines.pub public key.

NOTE that 'docker trust inspect' currently shows the signer's name and key ID (see below) ... but UNCLEAR how that ID can be compared against gwaines.pub .

$ docker trust inspect admin-2.cumulus.wrs.com:30093/wrcp-gwaines/hellokitty:v1.0
[
    {
        "Name": "admin-2.cumulus.wrs.com:30093/wrcp-gwaines/hellokitty:v1.0",
        "SignedTags": [
            {
                "SignedTag": "v1.0",
                "Digest": "204eb3b0cf6108a25c428bc5c9c82d76f9191dbe65de82426a4255214c1819ae",
                "Signers": [
                    "gwaines"
                ]
            }
        ],
        "Signers": [
            {
                "Name": "gwaines",
                "Keys": [
                    {
                        "ID": "135d386a1d25a9062e7faaeb9e0aa10f5880ab70b4390c17a186872e97e01789"
                    }
                ]
            }
        ],
...

**Comment by Jamstah** [comment]
I don't think this is a job for the registry - this seems like a feature request for the docker command line.
